### PR TITLE
Create change user screen

### LINKED ISSUE #2: Create change user screen

### PR DESCRIPTION
**Pull request by AndreeaMusca**

### CODE REVIEW

File: Game/Models/User.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+		public User(string username, int totalGames, int gamesWon, string profilePic)
+		{
+			Username = username;
+			TotalGames = totalGames;
+			GamesWon = gamesWon;
+			ProfilePic = profilePic;
+		}
+
+		[XmlElement("Username")]
```
Comment (daRealZoinks) [inline Game/Models/User.cs]: you dont need to write XmlElement to properties, they already get serialized

File: Game/ViewModels/ChangeUserViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	public class ChangeUserViewModel : ViewModelBase
+	{
+		private ObservableCollection<User> _users;
+		public ObservableCollection<User> Users
+		{
+			get { return _users; }
+			set
+			{
+				_users = value;
+				OnPropertyChanged("Users");
```
Comment (daRealZoinks) [inline Game/ViewModels/ChangeUserViewModel.cs]: write OnProepertyChanged() instead, the property gets automatically inserted as an argument

Comment (AndreeaMusca) [reply Game/ViewModels/ChangeUserViewModel.cs]: image

Comment (AndreeaMusca) [reply Game/ViewModels/ChangeUserViewModel.cs]: done

**Review by daRealZoinks — APPROVED** [review]
i like it, picasso

**Pull request merged**